processing/src/main/java/org/apache/carbondata/processing/loading/AbstractDataLoadProcessorStep.java

```diff
@@ -149,6 +153,22 @@ protected CarbonRowBatch processRowBatch(CarbonRowBatch rowBatch) {
    */
   protected abstract String getStepName();
 
+  /**
+   * This method registers all writer listeners and returns the listener
+   * @param rangeId bucketId
+   * @return
+   */
+  protected DataMapWriterListener getDataMapWriterListener(int rangeId) {
```

Comment:
remove the rangeId since it is always 0

Reply:
it i=will be zero in all places except  in a call from DataWriterProcessorStepImpl, in processRange function, so we can keep it i think

Reply:
then change the parameter name to bucketId